src/components/Test.js

```diff
@@ -0,0 +1,45 @@
+import React, { useState, useEffect } from 'react';
+import { fb } from '../lib/firebase';
+
+const Test = () => {
+  const db = fb.firestore();
+
+  const [test, setTest] = useState([]);
```

Comment:
This is a tiny naming thing, but I might name this:
```suggestion
  const [tests, setTests] = useState([]);
```
because it's an array of things, rather than just one thing. Since we won't use this going forward though, just something to consider in terms of how to name things.